Doc/library/typing.rst

```diff
@@ -1550,7 +1552,7 @@ These are not used in annotations. They are building blocks for creating generic
 
 .. data:: AnyStr
 
-   ``AnyStr`` is a :class:`constrained type variable <TypeVar>` defined as
+   ``AnyStr`` is a :ref:`constrained type variable <_typing-constrained-typevar>` defined as
```

Comment:
This link was incorrectly shown in monospace font. I took the opportunity to link to a more specific place.